app/routes/chatbot.js

```diff
@@ -44,58 +45,16 @@ function isCommand(incomingMessage, commandType) {
 }
 
 /**
- * Posts to chatbot route will find or create a Northstar User for the given req.body.phone.
- * Currently only supports Mobile Commons mData's.
+ * Check for required config variables.
  */
-router.post('/', (req, res) => {
-  const route = 'v1/chatbot';
-  stathat.postStat(`route: ${route}`);
-
-  const scope = req;
-  // Currently only support mobilecommons.
-  scope.client = 'mobilecommons';
-  // TODO: Define this in app.locals to DRY with routes/signups
-  scope.oip = process.env.MOBILECOMMONS_OIP_CHATBOT;
-  const agentViewOip = process.env.MOBILECOMMONS_OIP_AGENTVIEW;
-  scope.incoming_message = req.body.args;
-  scope.incoming_image_url = req.body.mms_image_url;
-
-  const logRequest = {
-    route,
-    profile_id: req.body.profile_id,
-    incoming_message: scope.incoming_message,
-    incoming_image_url: scope.incoming_image_url,
-  };
-
-  if (req.body.broadcast_id) {
-    scope.broadcast_id = req.body.broadcast_id;
-    logRequest.broadcast_id = scope.broadcast_id;
-  }
-
-  if (req.body.keyword) {
-    scope.keyword = req.body.keyword.toLowerCase();
-    logger.debug(`keyword:${scope.keyword}`);
-    logRequest.keyword = scope.keyword;
-  }
-
-  logger.info(logRequest);
-  newrelic.addCustomParameters({
-    incomingImageUrl: scope.incoming_image_url,
-    incomingMessage: scope.incoming_message,
-    keyword: scope.keyword,
-    mobileCommonsBroadcastId: scope.broadcast_id,
-    mobileCommonsMessageId: scope.body.message_id,
-    mobileCommonsProfileId: req.body.profile_id,
-  });
-
-  let configured = true;
-  // Check for required config variables.
+router.use((req, res, next) => {
```

Comment:
A question: won't this middleware be executed on every request, despite actual route?